recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - gdal 2.1.*
     - ghostscript
     - libnetcdf 4.4.*
```

Comment:
It is not `cmake`, sometimes `conda` gets "confused" resolving package conflicts, the issue lies with `libnetcdf `and `gdal`. Both dependencies must be available with latest `hdf5` before we re-triggerthe CIs here.